modules/sharedaddy/sharing-service.php

```diff
@@ -726,10 +726,19 @@ function sharing_display( $text = '', $echo = false ) {
 		}
 	}
 
+	/**
+	 * Filters the content of the Sharedaddy sharing links
+	 *
+	 * @since 3.8.0
+	 *
+	 * @param string $sharing_content Content of the Sharedaddy sharing links
+	 */
+	$sharing_display = apply_filters( 'sharing_display', $sharing_content );
```

Comment:
let's name this something more specific like `jetpack_sharing_display_markup`

h/t @jeherve